filters/builtin/inlinecontent_test.go

```diff
@@ -58,10 +59,10 @@ func TestInlineContentArgs(t *testing.T) {
 
 			c := f.(*inlineContent)
 
-			if c.text != test.expectedText {
+			if !reflect.DeepEqual(c.template, test.expectedTemplate) {
```

Comment:
not strictly for this PR, but I think we could just merge the two tests of TestInlineContentArgs and TestInlineContent into a single TestInlineContent one. On this line, we are comparing the template with DeepEqual opaque objects from another package, which in theory can cause various annoyances.

Reply:
...but if just checked that the right text is returned when the filter is applied after we merged two groups of tests, then we don't need to do this.

Reply:
Good point, fixed